src/chapters/typescript/compiling-typescript.rst

```diff
@@ -0,0 +1,64 @@
+Compiling TypeScript
+====================
+
+To work with TypeScript in our Node environment, we'll need to install the TypeScript
+module. In this section, you'll install the compiler and practice compiling and 
+running your TypeScript programs all from the terminal.
+
+To install Typescript, in your terminal, type
+
+.. sourcecode:: bash
+
+   $ npm install -g typescript
+
+Once you have TypeScript installed, you'll be able to run 
+
+.. sourcecode:: bash
+
+   $ tsc -v
+
+``tsc`` is is the TypeScript compiler and ``-v`` is the version flag to show us that
+we have TypeScript and it's compiler installed. 
+
```

Comment:
The tutorial could be placed in a separate section on the page.  Maybe something like Hello_World.ts?